vgscollect/src/main/java/com/verygoodsecurity/vgscollect/widget/VGSCardNumberEditText.kt

```diff
@@ -177,6 +180,26 @@ class VGSCardNumberEditText @JvmOverloads constructor(
         return getNumberDivider()
     }
 
+    /**
+     * Sets the symbol that will divide groups of digits in the card number before submit it.
+     * The divider has no impact on UI.
+     * 0000 0000 0000 0000
+     *
+     * @param char The divider symbol.
+     */
+    fun setOutputDivider(char: Char?) {
+        setOutputNumberDivider(char.toString())
```

Comment:
@Endorf I will suggest using `char?.toString()` because `char.toString()` will return string with value "null" if char == null, and as result if for example we will call `setOutputDivider(null)` we will see this message in logs `Divider for number can't be greater than 1 symbol`, same thing for `setDivider(char: Char?)`

Also I think we should check SSNEditText `setDivider/setOutputDivider` functions to.